packages/core/src/App/Containers/VerificationModal/verification-modal.scss

```diff
@@ -0,0 +1,359 @@
+                @include mobile {
+                    padding: 1.6rem 2.4rem;
+                }
+            }
+            .form-body {
+                @include desktop {
+                    z-index: 5;
+                }
+            }
+        }
+
+        .icon {
+            width: 128px;
+            height: 128px;
+            margin-top: 5.2rem;
+            margin-bottom: 2.6rem;
+
+            @include mobile {
+                width: 72px;
+                height: 72px;
+                margin-top: 2.6rem;
+            }
+        }
+
+        .external-dropdown {
+            .dc-dropdown-list {
+                z-index: 5;
+            }
+        }
+    }
+
+    &__redirection {
+        .dc-btn {
+            margin-top: 3.2rem;
+            height: 4rem;
+
+            @include mobile {
+                margin: 1.6rem 0;
+                padding: 1.6rem;
+                width: 100%;
+            }
+        }
+    }
+
+    &__country-text {
+        margin-bottom: 1.6rem;
+    }
+
+    &__text {
+        @include mobile {
+            width: 94%;
+            text-align: center;
+        }
+    }
+
+    &__dropdown-container {
+        margin-top: 1.6rem;
+    }
+
+    &__inner-container {
+        display: flex;
+        flex-direction: row;
+        justify-content: center;
+        align-items: center;
+        width: 100%;
+
+        @include mobile {
+            flex-direction: column;
+        }
+    }
+
+    &__header {
+        margin: 5.4rem 0 1.6rem;
+
+        @include mobile {
+            margin: 2.4rem 0 0.8rem;
+        }
+    }
+
+    &__footer.external-footer {
+        padding: unset;
+        position: unset;
+        top: unset;
+        bottom: unset;
+        border: none;
+        height: fit-content;
+        z-index: 4;
+    }
+
+    &__footer {
+        height: 8rem;
+        position: absolute;
+        bottom: 0;
+        left: 0;
+        display: flex;
+        width: 100%;
+        padding: 1.6rem 2.4rem;
+        align-items: center;
+        justify-content: flex-end;
+        z-index: 4;
+        border-radius: 0 0.4rem 0.4rem 0;
+        border-top: 0.1rem solid var(--general-section-1);
+        background-color: var(--general-main-1);
+
+        @include mobile {
+            display: flex;
+            flex-direction: row;
+            align-items: center;
+            justify-content: flex-end;
+            position: fixed;
+            bottom: 0;
+        }
+    }
+
+    &__footer-alert {
+        margin-right: auto;
+    }
+
+    &__submit-button {
+        @include mobile {
+            margin-right: 1.2rem;
+        }
+    }
+
+    &__fieldset {
+        width: 39.5rem;
+
+        @include mobile {
+            width: 94%;
+            margin: 0.4rem 0 3.2rem;
+        }
+
+        .document-dropdown {
+            margin: 0.4rem 0 3.2rem;
+        }
+
+        .country-dropdown {
+            min-height: 35.2rem;
+        }
+    }
+
+    &__fieldset-container {
```

Comment:
use nesting feature of scss here and for the rest of the changes in the file.
&__fieldset {
   &-container {
   ...
   }
   &-input {
   ...
   }
}